packages/sdk-js/src/handlers/server.ts

```diff
@@ -1,5 +1,30 @@
 import { AxiosInstance } from 'axios';
 
+export type ServerInfo = {
+	project: {
+		project_name: string;
+		project_logo: string | null;
+		project_color: string | null;
+		public_foreground: string | null;
+		public_background: string | null;
+		public_note: string | null;
+		custom_css: null;
+	};
+};
+
+export type OasData = {
```

Comment:
We should be able to reuse types from `openapi3-ts` in here

Reply:
Yeah, I didn't want to add dependency without checking with you. Added